docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -54,6 +54,12 @@ Setting `request_cache:true` on a query that creates a scroll (`scroll=1m`)
 has been deprecated in 6 and will now return a `400 - Bad request`.
 Scroll queries are not meant to be cached.
 
+==== Scroll queries cannot use `rescore`  anymore
+Including a rescore clause on a query that creates a scroll (`scroll=1m`)
+has been deprecated in 6 and will now return a `400 - Bad request`.  Allowing
+rescore on scroll queries would break the scroll sort.  In 6, the rescore
```

Comment:
maybe replace 6 with previous versions (only this second occurrence)? also when you mention the version can you specify also the minor version that is was deprecated in ?

Reply:
I've updated this language a bit.  I want to be careful here, because according to the original ticket, this was allowed at some time in the 5.x line.  It's not clear to me when exactly we stopped supporting rescoring scroll queries.

Reply:
Makes sense to me!